### PR TITLE
Update mmio table

### PR DESCRIPTION
**Pull request by dehanj**

### CODE REVIEW

File: hugo/content/protocol/_index.md [hunk 1]
```diff
@@ -149,8 +149,7 @@ The commands look like this:
 
 | *Name*           | *Size (bytes)* | *Comment*                                |
 |------------------|----------------|------------------------------------------|
-| Header           | 1              | Framing protocol header including length |
-|                  |                | of the rest of the frame.                |
+| Header           | 1              | Framing protocol header including length of the rest of the frame. |
```
Comment (mchack-work) [inline hugo/content/protocol/_index.md:152]: Why the change of line break here?

Comment (dehanj) [reply hugo/content/protocol/_index.md:152]: Because it is shown as a separate row on the [website](https://dev.tillitis.se/protocol/#firmware-protocol).

**Review by secworks — APPROVED** [review]

**Comment by dehanj** [comment]
Did a rebase on main.
